config/http/notifications/streaming-http/http.json

```diff
@@ -0,0 +1,83 @@
+{
+  "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^7.0.0/components/context.jsonld",
+  "@graph": [
+    {
+      "comment": "Path prefix used by streaming HTTP receiveFrom endpoints",
+      "@id": "urn:solid-server:default:variable:streamingHTTPReceiveFromPrefix",
+      "valueRaw": ".notifications/StreamingHTTPChannel2023/"
```

Comment:
In the request handler, I'm using
```js
  const topic = operation.target.path.replace(this.pathPrefix, '')
```
And in metadata writer:
```js
  const resourcePath = input.metadata.identifier.value.replace(this.baseUrl, '');
  const receiveFrom = `${this.baseUrl}${this.pathPrefix}${resourcePath}`;
```
If I use an InteractionRoute, it looks like I still need to pass `urn:solid-server:default:variable:baseUrl` to get the `pathPrefix` I need from `route.getPath()`. Is there a simpler way to do it?

Reply:
No you're right, I didn't pay enough attention to how this gets used. It's a middle part of an URL which makes it a bit more complicated. This might indeed be the best solution if this is how the URLs get formatted. It's just unfortunate that there will be some duplication then in the strings that are used.